dotcom-rendering/src/model/extract-nav.ts

```diff
@@ -97,5 +119,6 @@ export const extractNAV = (data: FENavType): NavType => {
 			  }
 			: undefined,
 		readerRevenueLinks: buildRRLinkModel(data),
+		selectedPillar: 'news',
```

Comment:
I could probably break this into `NavType` and `NavTypeWithSelectedPillar` but it felt like it didn't express all that much.